pkg/controller/client/auth.go

```diff
@@ -38,7 +41,13 @@ func (ac *AuthClient) Init(ctx context.Context, key string) (string, error) {
 	authDO := struct{ Key string }{key}
 	authDI := auth.JwtDO{}
 
-	err := ac.reqClient.Send(ctx, "/init", "POST", authDO, &authDI)
+	tokenPath := cmd.InsprOptions.Token
+	err := ioutil.WriteFile(tokenPath, []byte(""), os.ModePerm)
+	if err != nil {
+		return "", err
+	}
+
+	err = ac.reqClient.Send(ctx, "/init", "POST", authDO, &authDI)
```

Comment:
Might be a good idea to change "POST" to http.MethodPost

Reply:
Yess